README.md

```diff
@@ -175,13 +225,13 @@ Example usage
 
 ## Inclint (internal)
 
-This action should not probably used by workflows, rather by other actions, that implement linter tests for given languages. The action allows storing amount of linter errors for given branch in AWS S3 and comparing amount of linter errors results during PRs. It ensures, that amount of linter errors is incrementally decreased with time, by enforcing small improvement with every PR.
+This action should not probably used directly by workflows, rather by other actions, that implement linter tests for given languages. The action allows storing amount of linter errors for given branch in AWS S3 and comparing amount of linter errors results during PRs. It ensures, that amount of linter errors is incrementally decreased with time, by enforcing small improvement with every PR.
```

Comment:
https://dictionary.cambridge.org/grammar/british-grammar/amount-of-number-of-or-quantity-of

:D

Reply:
Thanks, fixed